config/locales/en.yml

```diff
@@ -150,6 +150,7 @@ en:
           other: "%{count} errors prohibited this %{model} from being saved"
         # The variable :count is also available
         body: "There were problems with the following fields:"
+        body: "There were problems with the following fields:"
```

Comment:
Duplicate?

Reply:
looks like it, but I'm not sure how, must have been a finger slip